### PR TITLE
Align page titles with Actions menu labels used to navigate to them

### PR DESCRIPTION
**Pull request by rhamilto**
After:
screen shot 2019-01-04 at 9 03 10 am

### CODE REVIEW

**Comment by spadgett** [comment]
jenkins rebuild

File: frontend/public/components/storage/create-pvc.tsx [hunk 1]
```diff
@@ -258,7 +258,7 @@ class CreatePVCPage extends React.Component<CreatePVCPageProps, CreatePVCPageSta
   state = {
     error: '',
     inProgress: false,
-    title: 'Create Storage',
+    title: 'Create Persistent Volume Claim',
```
[the remark this answers is not in the document]
Comment (rhamilto) [reply frontend/public/components/storage/create-pvc.tsx]: Updated to only include Image Pull Secret

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by rhamilto** [comment]
jenkins rebuild

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**